client/less/map.less

```diff
@@ -73,27 +82,100 @@
     display: flex;
     justify-content: flex-end;
 
-    padding-bottom: 0.7em;
-
-    background: linear-gradient(to bottom, #ffffff 0%, #ffffff 50%, rgba(255, 255, 255, 0) 100%);
-
+    padding-bottom: 5px;
+    padding-right:10px;
+    padding-top:5px;
+    background:#aaa;
     z-index: 100;
 }
 
 .map-fixed-header {
   position: fixed;
   background: white;
-  padding-top:15px;
+  padding-top: 15px;
   width: 100%;
   z-index: 1;
   left: 0;
   top: 0;
 }
+.map-fixed-header p {
+  margin: 10px 0 20px;
+}
+.map-fixed-header hr {
+  margin:30px 0;
+}
+.map-buttons button {
+  width:300px;
```

Comment:
Those buttons extend to whatever container they are in... courtesy of bootstrap.
There's absolutely no use or aesthetic gain to having those buttons changing size.